app/docs/enterprise/0.31-x/getting-started/quickstart.md

```diff
@@ -45,8 +48,10 @@ By default Kong listens on the following ports:
 - `:8443` on which Kong listens for incoming HTTPS traffic. This port has a
   similar behavior as the `:8000` port, except that it expects HTTPS
   traffic only. This port can be disabled via the configuration file.
-- `:8001` on which the [Admin API][API] used to configure Kong listens.
+- `:8001` on which the [Admin API][API] listens.
 - `:8444` on which the [Admin API][API] listens for HTTPS traffic.
+- `:8002` on which the [Admin GUI][GUI] listens.
+- `:8445` on which the [Admin GUI][GUI] listens for HTTPS traffic.
```

Comment:
Should we add another point which is "If you enable the Kong Developer Portal, `portal=on`, then Kong will listen to :8003 and :8004. I could see both arguments a) this is a quickstart, don't bog down tutorial with developer portal, or b) give higher visibility to dev portal

Reply:
Done, please review @darrenjennings